Tests/SwiftParserTest/translated/InitDeinitTests.swift

```diff
@@ -125,11 +125,11 @@ final class InitDeinitTests: ParserTestCase {
     assertParse(
       """
       struct FooStructDeinitializerA {
-        deinit 1️⃣x
+        1️⃣deinit x
       }
       """,
       diagnostics: [
-        DiagnosticSpec(message: "deinitializers cannot have a name", fixIts: ["remove 'x'"])
+        DiagnosticSpec(message: "deinitializers cannot have name", fixIts: ["remove 'x'"])
```

Comment:
Hmm, this isn’t grammatically correct anymore. I think we need the `a` in case the first element in the list is `name` or `return clause`.